elastic/kibana
# [maps] support null description in saved object

Part of https://github.com/elastic/kibana/issues/162069

> This happens in maps too and possibly other SOs that have null description (not sure if it were ever possible to have null description in maps.) It seems from the SOs that Bhavya sent that it was possible for visualizations. With this schema change the bug if fixed https://github.com/elastic/kibana/pull/162106

> Not sure if it makes sense to also do it in maps. @nreese wdyt?

Add support for null description in maps saved object to be consistent with visualize saved objects.

### test instructions
1) Install web logs sample data set
2) export map
3) manually change ndjson, changing `description:""` to `description:null`
4) reimport map saved object
5) verify map can be opened
